util/cron/test-perf.cray-cs.ofi.bash

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Run performance tests with comm=ofi on a cray-cs system.
+
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs'
+
+source $CWD/common-perf.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs.ofi"
+
+module load gcc
+module load python/2.7.6
+module load pmix
+
+export CHPL_HOST_PLATFORM=cray-cs
+export CHPL_COMM=ofi
+export CHPL_COMM_OFI_OOB=slurm-pmi2
+export SLURM_MPI_TYPE=pmix
+export CHPL_RT_MAX_HEAP_SIZE=83G
+export CHPL_LAUNCHER=slurm-srun
+export CHPL_LAUNCHER_PARTITION=bdw18
+export CHPL_TARGET_CPU=broadwell
+
+source /cray/css/users/chapelu/setup_libfabric.bash || return 1
+
+nightly_args="${nightly_args} -no-buildcheck"
+
+perf_args="-performance-description ofi -performance-configs gn-ibv-large,gn-ibv-fast:v,gn-mpi,ofi:v"
+perf_args="${perf_args} -performance -perflabel ml- -numtrials 1 -startdate 11/20/19"
```

Comment:
I think we want the startdate to match across the gasnet and ofi perf configs too. We end up generating and syncing the graphs for each configuration so the graph options should match.

Reply:
We should probably stuff all the perf_args except for `--numtrials` into a `common-perf-craycs.bash` file so they stay in sync.

Reply:
In a follow-up PR I'll make the startdate match and create `common-perf-craycs.bash`.